rain_api_core/urs_util.py

```diff
@@ -294,8 +294,19 @@ def do_login(args, context, cookie_domain=''):
         else:
             redirect_to = get_base_url(context)
 
+        if 'user_groups' not in user_profile:
```

Comment:
Wasn't there an issue where 'user_groups' was 'False' and that breaking things? Was that resolved elsewhere?

Reply:
I remembered something like that. Thought it was that there was no 'user_groups' key in dict. I'll build in handling in case it's set to False somehow